src/SFA.DAS.EmployerAccounts.MessageHandlers/EventHandlers/SignedAgreementEventHandler.cs

```diff
@@ -25,6 +31,16 @@ public async Task Handle(SignedAgreementEvent message, IMessageHandlerContext co
                 message.CohortCreated,
                 message.UserName,
                 message.UserRef.ToString()));
+
+            try
+            {
+                await _mediator.Send(new SignAccountAgreementCommand(message.AccountId, message.AgreementVersion, message.AgreementType.ToString()));
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
Shouldn't the exception be logged to Kibana?

Reply:
yes, this is just some local debug code I left in, will remove